css-advance/electives.css

```diff
@@ -0,0 +1,108 @@
+.page-content {
+    font-family: 'Times New Roman', Times, serif;

```

Comment:
For all text on the screen that is not button the font that you are using is a serif type of font. In the example all fonts are sans-serif type of fonts. Please change, as it is affecting the readability of the content. More information on the topic: https://medium.com/@milindsatam/what-is-serif-and-sans-serif-and-what-is-the-difference-between-them-bc345858374b